### PR TITLE
Correction for fixing request and response methods

### PR DESCRIPTION
**Pull request by sidharthanup**
Correction for fixing the request and response for Methods: GeoReplicationConfigGet and GeoReplicationConfigSet

Updates https://github.com/gluster/glusterd2/issues/584

### CODE REVIEW

**Comment by prashanthpai** [comment]
add to whitelist

**Comment by prashanthpai** [comment]
The `endpoints.md` is auto-generated and not meant to be hand edited. This needs a change in code

**Comment by vpandey-RH** [comment]
@sidharthanup You will need to edit the API definition in init.go file which seems to mention a request type for this API although there is no need. And then regenerate the endpoints.md.

File: doc/endpoints.md [hunk 1]
```diff
@@ -35,18 +35,18 @@ VolfilesGenerate | POST | /volfiles | [](https://godoc.org/github.com/gluster/gl
...
-SnapshotStatus | GET | /snapshots/{snapname}/status | [](https://godoc.org/github.com/gluster/glusterd2/pkg/api#) | [SnapStatusResp](https://godoc.org/github.com/gluster/glusterd2/pkg/api#SnapStatusResp)
-SnapshotDelete | DELETE | /snapshots/{snapname} | [](https://godoc.org/github.com/gluster/glusterd2/pkg/api#) | [](https://godoc.org/github.com/gluster/glusterd2/pkg/api#)
-SnapshotConfigGet | GET | /snapshots/config | [](https://godoc.org/github.com/gluster/glusterd2/pkg/api#) | [](https://godoc.org/github.com/gluster/glusterd2/pkg/api#)
-SnapshotConfigSet | POST | /snapshots/config | [](https://godoc.org/github.com/gluster/glusterd2/pkg/api#) | [](https://godoc.org/github.com/gluster/glusterd2/pkg/api#)
-SnapshotConfigReset | DELETE | /snapshots/config | [](https://godoc.org/github.com/gluster/glusterd2/pkg/api#) | [](https://godoc.org/github.com/gluster/glusterd2/pkg/api#)
+SnapshotCreate | POST | /snapshot | [](https://godoc.org/github.com/gluster/glusterd2/pkg/api#) | [](https://godoc.org/github.com/gluster/glusterd2/pkg/api#)
```
Comment (Madhu-1) [inline doc/endpoints.md]: `/snapshot` is now modified as `/snapshots` I think we are using an old gd2 code, please pull the latest code.

**Comment by Madhu-1** [comment]
@sidharthanup commit message contains 2 `sign-off` please update it

**Comment by sidharthanup** [comment]
@Madhu-1 Done

**Comment by atinmu** [comment]
Looks good to me.

**Pull request merged**